Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -478,7 +479,7 @@ void Adjuster::adjust(RenderStyle& style, const RenderStyle* userAgentAppearance
         // of the value of the position property, with one exception: as for boxes in CSS 2.1, outer ‘svg’ elements
         // must be positioned for z-index to apply to them.
         if (element && element->document().settings().layerBasedSVGEngineEnabled()) {
-            if (auto* svgElement = dynamicDowncast<SVGElement>(element); svgElement && svgElement->isOutermostSVGSVGElement())
+            if (isOutermostSVGElement(element))
```

Comment:
Not new but we're doing an unnecessary null check of `element` here.

Reply:
Fixed, but I was not able to reuse the static so there's some minor duplication.